data/json/recipes/recipe_food.json

```diff
@@ -59,6 +59,42 @@
   ]
 },{
   "type" : "recipe",
+  "result": "salt",
+  "id_suffix": "from_mineral",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_OTHER",
+  "skill_used": "cooking",
+  "difficulty": 0,
```

Comment:
If the recipe is 'break up the salt and clean it with some water', then that should be fine for autolearn.